lib-sql/tables.sql

```diff
@@ -244,6 +244,25 @@ CREATE UNIQUE INDEX idx_postcode_id ON location_postcode USING BTREE (place_id)
 CREATE INDEX idx_postcode_geometry ON location_postcode USING GIST (geometry) {{db.tablespace.address_index}};
 GRANT SELECT ON location_postcode TO "{{config.DATABASE_WEBUSER}}" ;
 
+-- Table to store location of entrance nodes
+DROP TABLE IF EXISTS place_entrance;
+CREATE TABLE place_entrance (
```

Comment:
We are entering nitpicking territory here but I'd prefer to name this table `placex_entrance` now as it has come out as an extension to `placex` in the end.